core/templates/pages/library-page/search-bar/search-bar.component.html

```diff
@@ -58,20 +58,20 @@
         </li>
       </ng-container>
       <li ngbDropdownItem *ngIf="selectionDetails.categories.numSelections > 1">
-        <a (click)="deselectAll('categories')" class="dropdown-item"><i>Deselect All</i></a>
+        <a tabindex="0" (keydown.enter)="deselectAll('categories')" (click)="deselectAll('categories')" class="dropdown-item"><i>Deselect All</i></a>
       </li>
       <hr *ngIf="selectionDetails.categories.numSelections > 0" class="oppia-search-bar-hr">
       <ng-container *ngFor="let item of selectionDetails.categories.masterList">
         <li ngbDropdownItem *ngIf="!selectionDetails.categories.selections[item.id]">
-          <a (click)="toggleSelection('categories', item.id)" *ngIf="!first && !last" class="dropdown-item">
+          <a tabindex="0" (keydown.enter)="toggleSelection('categories', item.id)" (click)="toggleSelection('categories', item.id)" *ngIf="!first && !last" class="dropdown-item">
             <span class="e2e-test-deselected">{{ item.text | translate }}</span>
             <i *ngIf="selectionDetails.categories.selections[item.id]" class="fas fa-check float-right oppia-search-bar-category-selection-symbol"></i>
           </a>
-          <a (click)="toggleSelection('categories', item.id)" *ngIf="first" (keydown)="onMenuKeypress($event, 'category', {shiftTab: ACTION_CLOSE})" class="dropdown-item">
+          <a tabindex="0" (keydown.enter)="toggleSelection('categories', item.id)" (click)="toggleSelection('categories', item.id)" *ngIf="first" (keydown)="onMenuKeypress($event, 'category', {shiftTab: ACTION_CLOSE})" class="dropdown-item">
```

Comment:
Ditto

Reply:
Not done? (break this line onto multiple lines.

Reply:
Done